src/main/java/Duke.java

```diff
@@ -52,6 +53,22 @@ public static void main(String[] args) {
                         count2++;
                     }
                     System.out.println("    ____________________________________________________________");
+
+                }else if (userInput.contains("delete")){
+                    content2 = userInput.substring(7);
+                    count4 = Integer.parseInt(content2) - 1;
+                    if(count4 == count){
+                        count = count - 1;
```

Comment:
You can create an ArrayList of Task to store the Task objects instead, and you can obtain the ArrayList size by using .getSize(), then you wouldn't need to create so many count variables to keep track of the number of Task.